sprinter/tests/test_environment.py

```diff
@@ -240,6 +241,17 @@ def test_source_to_target_config(self):
             eq_(environment.target.get('config', 'my_custom_value'), 'foo')
             eq_(environment.target.get('config', 'non_custom_value'), 'baz')
 
+    # def test_source_to_target_with_changed_default_config(self):
+    #     """ On an update, inputs with changed defaults in target should not have their values preserved in config """
+
+    #     with MockEnvironment(test_input_source, test_changed_input_default_target) as environment:
+    #         environment.directory = Mock(spec=environment.directory)
+    #         environment.directory.root_dir = "/tmp/"
+    #         environment.directory.new = False
+    #         environment.update()
+    #         eq_(environment.target.get('config', 'my_custom_value'), 'faa')
+    #         eq_(environment.target.get('config', 'non_custom_value'), 'baz')
```

Comment:
This flow prompts, but I'm not sure how to handle the interaction of submitting the prompt in the test. Maybe you could point me to an example.